website/settings/defaults.py

```diff
@@ -1967,6 +1967,8 @@ class CeleryConfig:
 STORAGE_LIMIT_PUBLIC = 50
 STORAGE_LIMIT_PRIVATE = 5
 
+GBs = 10 ** 9
+GiBs = 1024 ** 3
```

Comment:
Why are we using GBs in some places and GiBs in others?

Reply:
It depends on the context, some places GiBs were being used and just labeled GBs I went through the repo making sure the distinction was explicit.

Reply:
But _shouldn't_ we be using the same measure in different places?

Reply:
Well I can't with any certainty what should used in `scripts/osfstorage/usage_audit.py` or `osf/management/commands/export_user_account.py.` I just didn't want to change the behavior. The storage usage cap must be is GBs, because the `storage_usage` field is stored in GBs, because our size info is in GBs, but if you are just running a management command or script, I have to assume the behavior should just be maintained regardless of unit, I'm just making it explicit.

Reply:
If all the new things are GiBs, then could we just settle on GBs throughout the codebase?

Reply:
Well i just ran it through travis with no test fails so LGTM.